### DISCUSSION TITLE
Name for the app

### DISCUSSION THREAD

**Discussion by med4kat** [Nerds-Who-Code/Mental-Health-Tracker#6] [General]
We (@xintamosaik and @med4kat) went through the list of names suggested. Most of the names are not available (other companies own the name or name is used for example emoxi is medications). We came up with follow reduced list:

1. MentraTrack
2. <s>StressTrack</s> as per @SherlockX9 discord comment "options 2 and 3 aren't only what it does though"
3. <s>AnxietyStat</s> as per @SherlockX9 discord comment "options 2 and 3 aren't only what it does though"
4. Feelsify
5. Enotional
6. Moodkeeps

Could please let me know if you would like to add any additional names to the list? If not I will create polls to vote for the name tomorrow. 

That would not be a final decision we can change the name any time later. But we need something for now.

Many thanks!

**Comment by steph-koopmanschap** [comment]
Random suggestions:

Menta
MentraTrack
StressTrack
Mentalis - The mental health tracker
AnxietyStat

**Reply by xintamosaik** [reply]
feels

**Reply by xintamosaik** [reply]
feelsafe

**Reply by xintamosaik** [reply]
feelsify

**Reply by med4kat** [reply]
nice list!

We would need to check those names. Can't use them if someone is using them already.

Who is available to check those names on web?

**Comment by nicolecarman** [comment]
My ideas:

MoodView
MoodDeck
MoodCare
MoodEase
Moodful
Moodly
MindfulMood
Emotionally
Feeln
Emoxi

These are all plays on the following words: mood, mindful, feelings, emotions. I did a preliminary check to see if these names were taken before jotting them down. Research still needs to be done on them, but my initial search came up clear.

**Reply by xintamosaik** [reply]
enotional, yournal, moodkeeps
